pkg/infradriver/ssh_infradriver.go

```diff
@@ -94,12 +94,12 @@ func NewInfraDriver(cluster *v2.Cluster) (InfraDriver, error) {
 	}
 
 	// initialize sshConfigs field
-	for _, host := range cluster.Spec.Hosts {
-		if err = mergo.Merge(&host.SSH, &cluster.Spec.SSH); err != nil {
+	for i, host := range cluster.Spec.Hosts {
+		if err = mergo.Merge(&cluster.Spec.Hosts[i].SSH, &cluster.Spec.SSH); err != nil {
```

Comment:
What is the need for this code change?

Reply:
This part is not related to my commit, but it does cause lint errors, so I fixed it to pass CI. 

[https://github.com/sealerio/sealer/actions/runs/6611224807/job/17954685797](https://github.com/sealerio/sealer/actions/runs/6611224807/job/17954685797)

Reply:
In theory, this PR should only contain your own code, you can get the latest code through git rebase

Reply:
Okay, I'll drop these changes then.